modules/ROOT/pages/managing-queues.adoc

```diff
@@ -9,22 +9,29 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-CloudHub includes a *Queues* tab in the left navigation menu where you can view and control any Virtual Machine (VM) or Staged Event-Driven Architecture (http://en.wikipedia.org/wiki/Staged_event-driven_architecture[SEDA]) queues within the flows of your deployed applications:
+You can view and control any Virtual Machine (VM) or Staged Event-Driven Architecture (http://en.wikipedia.org/wiki/Staged_event-driven_architecture[SEDA]) queues within the flows of your deployed applications from the *Queues* tab in CloudHub.
 
-[NOTE]
-Queues are a feature that is currently only available for applications deployed to CloudHub workers. They are not available for applications that you deploy via the Runtime Manager to local servers. See xref:deployment-strategies.adoc[Deployment Strategies] for more details.
+Queues are currently only available for applications deployed to CloudHub workers. They are not available for applications that you deploy with the Runtime Manager to local servers. See xref:deployment-strategies.adoc[Deployment Strategies] for more details.
```

Comment:
Queues are available only for applications deployed to....
Queues are not available for applications deployed to on-premises servers.